### PR TITLE
Improve coverage for Intl Locale Info proposal

### PR DESCRIPTION
**Pull request by jugglinmike**
/cc @ryzokuken @FrankYFTang @Ms2ger

### CODE REVIEW

**Review by rwaldron — APPROVED** [review]
Approved, but one other person from @jugglinmike's cc list should take a look before this is merged

**Review by ryzokuken — APPROVED** [review]
LGTM, thanks @jugglinmike

**Comment by rwaldron** [comment]
Thanks @ryzokuken!

**Pull request merged**
